src/Tree.hpp

```diff
@@ -135,6 +140,30 @@ namespace RRT
 
 
         /**
+         * A coefficient that determines how much we want to expand our stepsize when adaptive stepsize control.
+         */
+        int ascLimit() const {
+            return _ascLimit;
+        }
+        void setASCLimit(int lim) {
+            _ascLimit = lim;
+        }
+
+
+        /**
+         * Whether or not the tree is to run with adaptive stepsize control.
+         */
+        bool isDynamic() const {
+            return _isDynamic;
+        }
+        void setASC(int checked) {
```

Comment:
If I remember correctly, the documentation had three possible values for checked. 0 for unchecked, 1 for partially unchecked, 2 for checked.